to-gatsby/.env.development

```diff
@@ -0,0 +1 @@
+GA_MEASUREMENT_ID=UA-41425441-7
```

Comment:
*technically* you definitely shouldn't check these IDs into CI, even though it's all of 30 second's work to scrape them from the webpage itself

Reply:
Yeah, fair enough. I'm trying to optimize for "this just runs when you do `yarn start`. Guess I could also include a fake mid which would still let you see the GA traffic in the network tab, but wouldn't put any real ids into source control.